src/oop/oop2.py

```diff
@@ -4,19 +4,44 @@
 # object is constructed.
 
 class GroundVehicle():
-    def __init__(self, num_wheels):
+    def __init__(self, num_wheels = 4):
         self.num_wheels = num_wheels
 
+    def __repr__(self):
+        print(f"This vehicle has " + str(self.num_wheels) +  " wheels.")   
+
+    def drive(self):
+        return "vroooom"    
+
     # TODO
 
 
+## When I print these I get a third print in the middle of them that says None, what does that refer to?
+gv = GroundVehicle(6)
+print(gv.__repr__())
```

Comment:
Instead of this try just doing 
```suggestion
print(gv)
```